doc/nrf/ug_nrf52.rst

```diff
@@ -0,0 +1,144 @@
+.. _ug_nrf52:
+
+Working with nRF52
+##################
+
+The |NCS| provides support for developing on all nRF52 Series devices and contains board definitions for all development kits and reference design hardware.
+
+Introduction
+************
+
+The nRF52 Series of System-on-Chip (SoC) devices embed a powerful yet low-power Arm Cortex-M4 processor with our industry leading 2.4 GHz RF transceivers.
+All of the nRF52 Series SoCs have support for Bluetooth 5 features, in addition to multi-protocol capabilities.
+
+See `nRF52 Series`_ for the technical documentation on the nRF52 Series chips and associated kits.
+
+Supported boards
+================
+
+All supported boards have a PCA number, which is used to uniquely identify each board.
+The build system uses a combination of device part code and PCA number to identify a board name in the format ``nRF52xxx-PCAxxxxx``.
+For example, for the :ref:`nRF52840 Development Kit (PCA10056) <zephyr:nrf52840_pca10056>`, the board name is ``nRF52840-PCA20056``, and you must use ``nrf52840_pca10056`` as target when building.
+
+Devices in the nRF52 Series are supported by these boards in the Zephyr open source project and in |NCS|.
+Refer to the PCA number for building projects for these boards and find a link to their descriptions.
+
+.. list-table::
+   :header-rows: 1
+
+   * - DK
```

Comment:
Suggestions:
- A column 'Device' could make it clearer for new customers which DK is used for which SoC. Also, someone looking for nRF52832 won't find it in the table (only after following the link to Zephyr).
- A note below the table referring to using nRF52840 DK for nRF52811 and nRF52 DK for nRF52810, explaining that the 'bigger' chip is used to emulate the functionality of the 'smaller', similar to what we had in the old SDK: https://infocenter.nordicsemi.com/topic/sdk_nrf5_v16.0.0/nrf52811_user_guide.html

Reply:
Leave as is because the title is supported boards not supported chips.